core/src/main/java/io/camunda/migrator/config/MigratorAutoConfiguration.java

```diff
@@ -147,7 +147,7 @@ public ProcessEngineConfigurationImpl processEngineConfiguration() {
     var config = new SpringProcessEngineConfiguration();
     config.setDataSource(c7DataSource);
     config.setTransactionManager(c7TransactionManager);
-    config.setHistory(HISTORY_AUTO);
+    config.setHistory(HISTORY_FULL);
```

Comment:
This was set to `AUTO` since the Data Migrator usually connects to already existing C7 databases that have been configured for history level. If we set this to `FULL`, the Data Migrator will fail if it detects a history level other than `FULL`. If you need to configure `FULL` for testing purposes, maybe create a `processEngineConfiguration` bean for testing only and mark this bean as conditional.